packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -474,13 +475,28 @@ export class Table extends Resource {
   }
 
   /**
-   * Permis an IAM principal all stream data read operations for this
+   * Permits an IAM Principal to list all DynamoDB Streams.
+   * @param grantee The principal (no-op if undefined)
```

Comment:
I would add an empty line before `@param` (makes the docs more readable and less cramped).